bin/yaml/racket.yaml

```
@@ -0,0 +1,13 @@
+compilers:
+  racket:
+    type: script
+    check_exe: bin/racket --version
+    dir: racket-{name}
+    fetch:
+      - https://download.racket-lang.org/releases/{name}/installers/racket-{name}-x86_64-linux-cs.sh racket-{name}.sh
+    script: |
+      sh racket-{name}.sh --in-place --dest ./racket-{name}
```

I forgot we're trying to get everything into subdirectories, so something like this:

```suggestion
    dir: racket/racket-{name}
    fetch:
      - https://download.racket-lang.org/releases/{name}/installers/racket-{name}-x86_64-linux-cs.sh racket-{name}.sh
    script: |
      sh racket-{name}.sh --in-place --dest ./racket/racket-{name}
```

(Note that it also needs an update in the main repo PR to match the new paths)

---

Ah, makes sense, I've updated this to match.